frontend/national-shooting-championships/index.html

```
@@ -0,0 +1,704 @@
+            <option value="gagan-narang">Gagan Narang</option>
+            <option value="rajyavardhan-rathore">Rajyavardhan Singh Rathore</option>
+            <option value="avani-lekhara">Avani Lekhara</option>
+            <option value="rahi-sarnobat">Rahi Sarnobat</option>
+            <option value="swapnil-kusale">Swapnil Kusale</option>
+            <option value="sarabjot-singh">Sarabjot Singh</option>
+            <option value="anjali-bhagwat">Anjali Bhagwat</option>
+          </select>
+        </div>
+
+        <div id="shooter-profile" class="shooting-boxer-profile" aria-hidden="true" style="display: none;">
+          <img id="shooter-profile-img" src="" alt="" loading="lazy">
```

HTMLHint (1.9.2)

[error] The attribute [ src ] of the tag [ img ] must have a value.

(src-not-empty)